analyzers/src/SonarAnalyzer.CSharp/Rules/GenericReadonlyFieldPropertyAssignment.cs

```diff
@@ -59,7 +59,8 @@ protected override void Initialize(SonarAnalysisContext context)
                 SyntaxKind.OrAssignmentExpression,
                 SyntaxKind.LeftShiftAssignmentExpression,
                 SyntaxKind.RightShiftAssignmentExpression,
-                SyntaxKindEx.CoalesceAssignmentExpression);
+                SyntaxKindEx.CoalesceAssignmentExpression,
+                SyntaxKindEx.GreaterThanGreaterThanGreaterThanEqualsToken);
```

Comment:
Use. At this point, registering to the token or the UnsignedRightShiftAssignmentExpression might give back the same syntax node but that might change in the future. This is the correct way to do it. (In a weird case that token can appear in other constructs too.)
```suggestion
                SyntaxKindEx.UnsignedRightShiftAssignmentExpression);
```